src/templates/components/fonts/EgyptianWeb.css

```diff
@@ -0,0 +1,12 @@
+@font-face {
+	font-family: 'Guardian Egyptian Web';
+	src: url('https://assets.guim.co.uk/static/frontend/fonts/guardian-headline/noalts-not-hinted/GHGuardianHeadline-Medium.woff2?http3=true')
+			format('woff2'),
+		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-headline/latin1-not-hinted/GHGuardianHeadline-Medium.woff?http3=true')
+			format('woff'),
+		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-headline/latin1-not-hinted/GHGuardianHeadline-Medium.ttf?http3=true')
+			format('truetype');
+	font-weight: 500;
+	font-style: normal;
+	font-display: swap;
+}
```

Comment:
Is this taken from https://github.com/guardian/fonts?

Reply:
Ah good spot - this font was actually in the Headline file but the legacy template had it under a different name. I've now deleted this font face and am using the one from the Headline font file.